### ISSUE TITLE
Broken tests in 0.16.x branch

### ISSUE THREAD

**Issue by benjaoming** [learningequality/ka-lite#5169]
I've manually checked that the feedback form is displaying, however this test is still failing.

```
[INFO] [2016-06-17 18:26:48,722] django.request: HTTP Request /api/content/khan/undefined_entry_id/ - Response: 200
    Given I open some available content # kalite/distributed/features/steps/learn_content.py:14 1.368s
    Then I see a feedback form          # kalite/distributed/features/steps/content_rating.py:40
[INFO] [2016-06-17 18:26:48,876] django.request: HTTP Request /api/videolog/ - Response: 200
    Then I see a feedback form          # kalite/distributed/features/steps/content_rating.py:40 60.567s
      Traceback (most recent call last):
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1173, in run
          match.run(runner.context)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1589, in run
          self.func(context, *args, **kwargs)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_rating.py", line 42, in impl
          feedback_form_container = find_id_with_wait(context, RATING_CONTAINER_ID, wait_time=60)
        File "/home/ubuntu/ka-lite/kalite/testing/behave_helpers.py", line 222, in find_id_with_wait
          return _find_elem_with_wait(context, (By.ID, id_str), **kwargs)
        File "/home/ubuntu/ka-lite/kalite/testing/behave_helpers.py", line 267, in _find_elem_with_wait
          EC.presence_of_element_located(by)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/selenium/webdriver/support/wait.py", line 75, in until
          raise TimeoutException(message, screen, stacktrace)
      TimeoutException: Message: 
      Stacktrace:
          at <anonymous class>.FirefoxDriver.prototype.findElementInternal_ (file:///tmp/anonymous7166039749452953088webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
          at <anonymous class>.FirefoxDriver.prototype.findElement (file:///tmp/anonymous7166039749452953088webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10779)
          at <anonymous class>.DelayedCommand.prototype.executeInternal_/h (file:///tmp/anonymous7166039749452953088webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12661)
          at <anonymous class>.DelayedCommand.prototype.executeInternal_ (file:///tmp/anonymous7166039749452953088webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12666)
          at <anonymous class>.DelayedCommand.prototype.execute/< (file:///tmp/anonymous7166039749452953088webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12608)
```

And so is this

```
[INFO] [2016-06-17 18:28:08,719] django.request: HTTP Request /api/videolog/ - Response: 200
    Given I have filled out a feedback form # kalite/distributed/features/steps/content_rating.py:121 32.723s
      Assertion Failed: FAILED SUB-STEP: When I fill out the form
      Substep info: Traceback (most recent call last):
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1173, in run
          match.run(runner.context)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1589, in run
          self.func(context, *args, **kwargs)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_rating.py", line 116, in impl
          enter_star_ratings(context)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_rating.py", line 173, in enter_star_ratings
          rate_id(context, id_, val=val)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_rating.py", line 196, in rate_id
          raise Exception("Unable to enter rating for container with id '{id:s}'".format(id=id_))
      Exception: Unable to enter rating for container with id 'star-container-quality'
```

And this

```
[INFO] [2016-06-17 18:28:52,495] django.request: HTTP Request /api/videolog/ - Response: 200
    Given I have filled out a feedback form # kalite/distributed/features/steps/content_rating.py:121 33.895s
      Assertion Failed: FAILED SUB-STEP: When I fill out the form
      Substep info: Traceback (most recent call last):
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1173, in run
          match.run(runner.context)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1589, in run
          self.func(context, *args, **kwargs)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_rating.py", line 116, in impl
          enter_star_ratings(context)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_rating.py", line 173, in enter_star_ratings
          rate_id(context, id_, val=val)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_rating.py", line 196, in rate_id
          raise Exception("Unable to enter rating for container with id '{id:s}'".format(id=id_))
      Exception: Unable to enter rating for container with id 'star-container-quality'
```

And this
```
[INFO] [2016-06-17 18:29:40,970] django.request: HTTP Request /api/videolog/ - Response: 200
    Given I have filled out a feedback form # kalite/distributed/features/steps/content_rating.py:121 33.070s
      Assertion Failed: FAILED SUB-STEP: When I fill out the form
      Substep info: Traceback (most recent call last):
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1173, in run
          match.run(runner.context)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1589, in run
          self.func(context, *args, **kwargs)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_rating.py", line 116, in impl
          enter_star_ratings(context)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_rating.py", line 173, in enter_star_ratings
          rate_id(context, id_, val=val)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_rating.py", line 196, in rate_id
          raise Exception("Unable to enter rating for container with id '{id:s}'".format(id=id_))
      Exception: Unable to enter rating for container with id 'star-container-quality'
```

And this

```
[INFO] [2016-06-17 18:31:45,178] django.request: HTTP Request /api/topic_tree/khan - Response: 200
    Given I open some available content # kalite/distributed/features/steps/learn_content.py:14 1.541s
    Then I should see no alert          # kalite/distributed/features/steps/learn_content.py:31
[INFO] [2016-06-17 18:31:45,316] django.request: HTTP Request /api/content/khan/undefined_entry_id/ - Response: 200
    Then I should see no alert          # kalite/distributed/features/steps/learn_content.py:31 30.267s
      Traceback (most recent call last):
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1173, in run
          match.run(runner.context)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1589, in run
          self.func(context, *args, **kwargs)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/learn_content.py", line 35, in impl
          assert_no_element_by_css_selector(context, ".alert")
        File "/home/ubuntu/ka-lite/kalite/testing/behave_helpers.py", line 104, in assert_no_element_by_css_selector
          _assert_no_element_by(context, By.CSS_SELECTOR, css_value, wait_time)
        File "/home/ubuntu/ka-lite/kalite/testing/behave_helpers.py", line 85, in _assert_no_element_by
          EC.presence_of_element_located((by, value))
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/selenium/webdriver/support/wait.py", line 91, in until_not
          raise TimeoutException(message)
      TimeoutException: Message:

Link to your job: https://saucelabs.com/jobs/da19af73e85247429930b2dfda0ee2f7
Installed 0 object(s) from 0 fixture(s)
```

**Comment by benjaoming** [comment]
Tests work fin in `develop` after merging 0.16.x!

**Comment by benjaoming** [comment]
BDD tests fixed in #5215
